boards/riscv/longan_nano/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _longan_nano:
+
+Sipeed Longan Nano
+##################
+
+Overview
+********
+
+The Sipeed Longan Nano and Longan Nano Lite is an simple and tiny development board with
+an GigaDevice GD32VF103 SoC that based on N200 RISC-V IP core by Nuclie system.
+More information can be found on
+`Sipeed's website <https://longan.sipeed.com/en/>`_ and
+`Nuclei's website <https://www.nucleisys.com/download.php>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``logan_nano`` board configuration can be built as usual
+(see :ref:`build_an_application`) using the corresponding board name:
+
+.. zephyr-app-commands::
+   :board: logan_nano
+   :goals: build
+
+Flashing
+========
+
+In order to upload the application to the device, you'll need OpenOCD with
+GD32V support. Download the tarball for your OS from the
+`SiPEED longan nano download site
+<http://dl.sipeed.com/LONGAN/platformio/dl-packages/>`_ and extract it.
+
+The Zephyr SDK uses a bundled version of OpenOCD by default. You can
+overwrite that behavior by adding the
+``-DOPENOCD=<path/to/riscv-openocd/bin/openocd>`` and
+``-DOPENOCD_DEFAULT_PATH=<path/to/openocd/scripts>``
+parameter when building:
+
+.. zephyr-app-commands::
+   :board: longan_nano
+   :goals: build
+   :gen-args: -DOPENOCD=<path/to/riscv-openocd/bin/openocd> -DOPENOCD_DEFAULT_PATH=<path/to/openocd/scripts>
```

Comment:
Keep only `:gen-args: -DOPENOCD=<path/to/riscv-openocd/bin/openocd>`

Reply:
I'll apply it.